### ISSUE TITLE
[a11y Process Work] Internal Organization Request from CAIA - A11y

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#63340]
## About your team

- Team name: Sitewide CAIA
-	OCTO-DE product owner: @artsymartha68 
-	Product manager: @coforma-terry 
-	Designer: 
- Accessibility specialist: @coforma-jamie @eli-oat @sara-amanda @SarahKay8 @ohgreatkate 
- Content writer: 
-	Slack channel: #sitewide-content-accessibility-IA
 
### Describe what you need in a few sentences:
As I’ve been working my way through an a11y audit, there’s been a lot of confusion (mainly from me!) around what the expected deliverables are. And I’ve felt that throughout - what is expected from CAIA for This Thing, or That Thing? There’s a lot of Names Of Things, a lot of templates, and it’s hard to know what goes with what. As someone new to to the team, I’ve been spending a lot of time trying to wrap my brain around it, and it hasn’t always gone well.

As such, I want to propose that we pause - For a day? Not sure how to actually lay this out - get together as a team, and come up with a document that has:

- the names of all the types of things we do: audits (and types of audits), reviews (and types of reviews), etc
- a description (in plain language) of what each is for, when someone would ask for said type
- a list of what the CAIA staff member is responsible for, for each type
- links to the appropriate templates, both GH and personal use to keep track of stuff, for each type (where applicable)

This would be a team Source of Truth, laid out as simply as possible, sharable with anyone who needs to see it, able to be passed on to the revolving CAIA team members of the future.

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [x] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [x] I’d like to schedule time to talk about this.
- [ ] Let’s work async and meet as needed.

**Comment by eli-oat** [comment]
@coforma-jamie this is well aligned with some work I'm currently tackling. 

I need to get a wee bit further with that stuff, but once it is in the right spot I'll circle to this! 

In the meantime, I've assigned this to myself, too, so that I don't lose track of this! 

tl;dr of what I'm working on: 

- drafting an initial menu of offerings for CAIA accessibility
- this will include the services we can provide
- the tasks that need to be done for a given service
- documentation for how to perform those tasks
- where this relates, is that I also hope to define what deliverables each service generates

I'll rough something in to give us a starting point, and then we can get together to start refining and identifying questions and pain points.

**Comment by sara-amanda** [comment]
## CAIA Update 

@eli-oat  thank you for the Super Epic! 

@coforma-jamie in the meantime, if you need any clarity, happy to help if I can. If not, I can always hunt for the right person to ask, to assist. 

### Responses based on current resources

> - [x] the names of all the types of things we do: audits (and types of audits), reviews (and types of reviews), etc 
- [Refer to and please feel free to add to the CAIA access points noted.](https://docs.google.com/document/d/1_UZ17sHg_hz57xS1moaqfMvLW9fOT6nSX9sgkY_ZN34/edit#heading=h.12wo0zw4hsp1)
- **Here is what we have so far, as entry points for CAIA in our CAIA a11y guide and purpose doc:** 
    - Assistive Technology Research Support
    - Collaboration Cycle
    - Non-Collaboration Cycle
    - DST Pattern or Component Reviews
    - Slack Issues via Teams
    - a11y Help Channel Support
    - Office Hours
    - Defects and blockers - tickets
    - Kickoff Meetings - post intake ticket
    - Maintenance Audits
 
> - [x] a description (in plain language) of what each is for, when someone would ask for said type

- [Refer to the CAIA ticket templates section. I see you added the maintenance audit @coforma-jamie thank you!](https://docs.google.com/document/d/1_UZ17sHg_hz57xS1moaqfMvLW9fOT6nSX9sgkY_ZN34/edit#heading=h.vttxn8wqd6a4)
- There is also a description above each template in GitHub. Examples below.
    - I am not sure where to edit it, but we can take a look and make sure that matches any of our CAIA documentation. 
    - I think adding to the descriptions would be helpful to everyone. 
- Some templates we share, so we would need to make it a11y focused and not just CAIA in the descriptions.
    - **Example CAIA Template Description**: Issue: Assistive Technology Research Support Ticket (CAIA)
        - Request support for research with disabled Veterans using assistive technology. 
        - _This line appears on all descriptions: If this doesn't look right, [choose a different type.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose)_
    - **Example a11y Template:** Accessibility Maintenance Audit
        - Create an epic for a maintenance a11y audit outside of the collaboration cycle.
        - _This line appears on all descriptions: If this doesn't look right, [choose a different type.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose)_
> - [ ] a list of what the CAIA staff member is responsible for, for each type
- This sounds like @eli-oat 's menu he is working on. If you request a, you will receive b.
- Are we thinking we would add this to the ticket body to, in order to set expectations? 
> - [x] links to the appropriate templates, both GH and personal use to keep track of stuff, for each type (where applicable)
- These can be found in the folders at the top of the CAIA Core Slack Channel and the CAIA Accessibility Specialists Channel.
- [Our CAIA doc, in the accessibility template section](https://docs.google.com/document/d/1_UZ17sHg_hz57xS1moaqfMvLW9fOT6nSX9sgkY_ZN34/edit#heading=h.vttxn8wqd6a4) breaks down when to use what ticket, if that works as something we can expand on for this project, too. 

> - [ ] This would be a team Source of Truth, laid out as simply as possible, sharable with anyone who needs to see it, able to be passed on to the revolving CAIA team members of the future.
- A simplified GH page with this information would be ideal. I like the source of truth and we can date it with the date last updated. 
> 


## Epics
- We also have this [Epic] CAIA Accessibility Documentation #55956, that @joshkimux helped to create back in March, when I first started, here. 
- It is for a11y (and it includes CAIA but expands beyond CAIA).
- It has been added to your Super Epic @eli-oat , to connect it. 

cc: @coforma-terry

**Comment by sara-amanda** [comment]
@eli-oat @coforma-terry does this align with your process work you are both doing? Should we close this and consolidate with your work?

**Comment by coforma-jamie** [comment]
Over the past few months, we've made excellent progress on improving our internal processes, including:

- Creating and adjusting ticket templates
- Creating a process for AT research pilots, along with documentation for VFS teams
- Using tasks and GitHub Projects to track our work
- Creating a CAIA accessibility "menu" of services that teams can reference
- CAIA-wide: Removing "super-epics"
- CAIA-wide: Determining who assigns tickets for our different disciplines
- CAIA-wide: Collecting metrics on team member workload

We feel that we've sufficiently improved our processes (and will adjust in the future as needed), and are closing out this ticket.

**Comment by artsymartha68** [comment]
Amen! Thanks for all the improvements!
